src/core/store.js

```diff
@@ -0,0 +1,18 @@
+import {configureStore} from "@reduxjs/toolkit";
+import movieBrowserReducer from '../features/MoviesBrowser/movieBrowserSlice';
+import createSagaMiddleware from "redux-saga";
+import saga from "./saga";
+
+const sagaMiddleware = createSagaMiddleware();
+
+const store = configureStore({
+    reducer: {
+        movieBrowser: movieBrowserReducer,
```

Comment:
You can add this reducer to store.js which already exist

Reply:
Of course